package-lock.json

```diff
@@ -8312,6 +8312,11 @@
       "resolved": "https://registry.npmjs.org/fast-levenshtein/-/fast-levenshtein-2.0.6.tgz",
       "integrity": "sha1-PYpcZog6FqMMqGQ+hR8Zuqd5eRc="
     },
+    "fast-safe-stringify": {
```

Comment:
@skyhit No changes to `package.json`, so please remove changes to `package-lock.json`